Tests/GravityTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Orbitale\Component\ImageMagick\Tests;
+
+use Orbitale\Component\ImageMagick\Command;
+use Orbitale\Component\ImageMagick\ReferenceClasses\Gravity;
+
+class GravityTest extends AbstractTestCase
+{
+    /**
+     * @param string $gravity
+     *
+     * @dataProvider provideValidGravities
+     */
+    public function testGravity($gravity): void
+    {
+        $gravity = new Gravity($gravity);
+
+        $validatedGravity = $gravity->validate();
+
+        static::assertIsString($validatedGravity);
```

Comment:
```suggestion
```

Useless check too, since `->validate()`'s return type is `string`

Reply:
Fair enough, resolved in 8e5d443.